kim/kim-ldap/src/main/java/edu/arizona/kim/service/impl/UaEdsIdentityService.java

```diff
@@ -27,6 +36,12 @@
 public class UaEdsIdentityService extends LdapIdentityDelegateServiceImpl implements LdapIdentityService {
```

Comment:
To followup on this: UAF-2303 has been opened and approved to re-factor/re-write/re-wire a sane IdentityService hierarchy that will address all of these shortcomings. I am currently working said jira, so this will addressed shortly via that ticket. Thanks again for adding justification.